### PR TITLE
Updates the application status pattern. [Amends #3163]

### PR DESCRIPTION
**Pull request by humancompanion-usds**

### CODE REVIEW

**Comment by jeana-adhoc** [comment]
I updated the email screenshots as some things were updated in github but not in the figma file.

File: src/_patterns/help-users-to/stay-informed-of-their-application-status.md [hunk 1]
```diff
@@ -3,47 +3,103 @@ layout: pattern
...
-Review examples of patterns that exist today that help users to stay informed of their application status.
-
 ## Usage
 
 ### When to use this pattern
 
-**When users need to stay informed of their application status.** After users have submitted an application, keep them informed of their application status.
+* **Any application that takes data from a user and submits that data to a back-end system.** This pattern is widely applicable to any application or service that accepts users data and submits that data, synchronously or asynchronously, to a system of record. It is a requirement that we notify the user at critical touch points as their request is processed.
+
+### When not to use this pattern
```
Comment (danbrady) [inline src/_patterns/help-users-to/stay-informed-of-their-application-status.md:22]:
```suggestion

```
If there is no guidance, we can we remove the "when not to use" subtitle

File: src/_patterns/help-users-to/stay-informed-of-their-application-status.md [hunk 2]
```diff
@@ -3,47 +3,103 @@ layout: pattern
...
+3. **Providing a clear next step in the event of an error.**
+
+#### How to provide a clear next step in the event of an error
+
+In the event of an error, an application may determine the next logical step a user can take to remedy that error. At the very least users must be notified that they can contact the Call Center in the event of a submission failure.
 
 ### Components used in this pattern
 
 * [Alert]({site.baseurl}/components/alert)
 * [Card]({site.baseurl}/components/card)
```
Comment (danbrady) [inline src/_patterns/help-users-to/stay-informed-of-their-application-status.md:87]:
```suggestion
* [Alert]({{site.baseurl}}/components/alert)
* [Card]({{site.baseurl}}/components/card)
```
Links were broken. (Need additional brackets.)

File: src/_patterns/help-users-to/stay-informed-of-their-application-status.md [hunk 3]
```diff
@@ -3,47 +3,103 @@ layout: pattern
...
-* [Process list]({site.baseurl}/components/process-list)
\ No newline at end of file
+
+### Page templates available for this pattern
+
+[Email templates are available in VA Notify](https://notifications.va.gov/information/emails).
```
Comment (danbrady) [inline src/_patterns/help-users-to/stay-informed-of-their-application-status.md:91]: Is this link suppose to work?

**Review by danbrady — CHANGES_REQUESTED** [review]

File: src/_patterns/help-users-to/stay-informed-of-their-application-status.md (hunk 3 above)
Comment (jeana-adhoc) [reply src/_patterns/help-users-to/stay-informed-of-their-application-status.md:91]: You have to be logged in on the VA VPN/Citrix for it to work. Maybe we should include those instructions as well.

Comment (humancompanion-usds) [reply src/_patterns/help-users-to/stay-informed-of-their-application-status.md]: Added a note.

File: src/_patterns/help-users-to/stay-informed-of-their-application-status.md [hunk 4]
```diff
@@ -3,47 +3,103 @@ layout: pattern
...
+{% include component-example.html alt="Screen shots of 3 email templates." file="/images/patterns/help-users-to/stay-informed-of-their-application-status/email-templates.png" caption="Three templates for email notification touch points sent via VA Notify." class="x2" %}
 
-### 10-10ez application status on health care intro page
+### Status in MyVA
 
-{% include component-example.html alt="An example of the 10-10ez health care application status on intro page." file="/images/patterns/help-users-to/stay-informed-of-their-application-status/10-10ez-status-on-intro-page.png" caption="The 10-10ez Health Care application provides status to users if and when they return to the introduction page." class="x2" %}
+{% include component-example.html alt="Screen shots of 3 MyVA status cards." file="/images/patterns/help-users-to/stay-informed-of-their-application-status/myva-status-cards.png" caption="Three Card component variations used to convey status of a user's submission in MyVA for Benefits applications." reverse="true" %}
 
-### Claim status
+## How to design and build
```
[the remark this answers is not in the document]
Comment (humancompanion-usds) [reply src/_patterns/help-users-to/stay-informed-of-their-application-status.md:33]: Yeah, an intake ticket to point to would be great. But I can add that later, if available.

**Review by danbrady — APPROVED** [review]

File: src/_patterns/help-users-to/stay-informed-of-their-application-status.md [hunk 5]
```diff
@@ -3,47 +3,102 @@ layout: pattern
...
+{% include component-example.html alt="Screen shots of 3 MyVA status cards." file="/images/patterns/help-users-to/stay-informed-of-their-application-status/myva-status-cards.png" caption="Three Card component variations used to convey status of a user's submission in MyVA for Benefits applications." reverse="true" %}
+
+## How to design and build
+
+### Email notification touch points
+
+When a request reaches these 3 touch points the user must be notified via email: 
+
+1. **Submission in progress**: The status immediately after a user clicks "submit" on a form. This is the period between the submission and when that submission reaches a system of record. During this time, data submitted by the user may travel through several systems.
+  * **Not required for synchronous status.** This touch point is only necessary if the user's data submission is being handled asynchronously. If you can synchronously confirm that the user's data has reached the system of record immediately then you can skip this status notification.
```
Comment (babsdenney) [inline src/_patterns/help-users-to/stay-informed-of-their-application-status.md:40]: Will Designers know the difference between a synchronous and asynchronous status? I don't know if I'm completely sure how that is defined but I don't work with the forms.

**Pull request merged**